assets/script.js

```diff
@@ -0,0 +1,75 @@
+function main() {
+  // correctAnswer and currentAnswer used for game logic
+  const correctAnswer = ["button1", "button3", "button2"];
+  let currentAnswer = [];
+  const gameOutcomeEl = document.getElementById("gameOutcome");
+
+  // handles all the event listeners for the buttons
+  function buttonEvents() {
+    const button1El = document.getElementById("button1");
+    const button2El = document.getElementById("button2");
+    const button3El = document.getElementById("button3");
+
+    button1El.addEventListener("click", function () {
+      bringTheNoise("C4");
+      guessingGame("button1");
+    });
+    button2El.addEventListener("click", function () {
+      bringTheNoise("C5");
+      guessingGame("button2");
+    });
+    button3El.addEventListener("click", function () {
+      bringTheNoise("C6");
+      guessingGame("button3");
+    });
+  }
+  buttonEvents();
+
+  // Contains all the logic for the mini game
+  function guessingGame(button) {
+    // checks the users guess
+    // -- if correct the answer is added to the currentAnswer array
+    // -- if incorrect, the currentAnswer array is set back to an empty array and the user is informed they were wrong
+    if (correctAnswer[currentAnswer.length] === button) {
```

Comment:
Minor:
You could make this a lot more readable by doing the following:

```suggestion
    const incorrectAnswer = correctAnswer[currentAnswer.length] === button;
    if (incorrectAnswer) {
```

Reply:
it seems that @monarchwadia was ahead of me